### ISSUE TITLE
sprite_create_from_screen() seems broken in OpenGL1, OpenGL3.

### ISSUE THREAD

**Issue by sorlok** [enigma-dev/enigma-dev#794]
Unless I'm doing it wrong, the following function call fails to create a drawable sprite.

```
self.myspr = sprite_create_from_screen(32,8,100,100,false,false,false,0,0);
```

Sample GM:S project:
https://drive.google.com/file/d/0B1P7NepPcOslcFNtUC1FVkNQRFE/edit?usp=sharing

Can anyone confirm if this is working on DirectX? I am not 100% familiar with sprite_create_from_screen, and GM:S seems to have recently removed it (probably when they switched everything over to surfaces).

**Comment by TheExDeus** [comment]
Yes, It's in the calculation of the screen flip. Right now there is h-y but it needs to be screen_height-h-y. This means I need to add requirement for window_get_region_height_scaled() in those headers, which is awful, as we struggle to get less coupling.
My fixes: https://github.com/enigma-dev/enigma-dev/commit/6d2f9fc0958ec05c5e4205f39942e3ac82b07e77

**Comment by sorlok** [comment]
Thanks for working on this! I'll have a look at your fix and see how it affects the test cases I have.
